jdi-light/src/main/java/com/epam/jdi/light/elements/composite/WebPage.java

```diff
@@ -80,10 +82,10 @@
 
     public <T> Form<T> asForm() {
         return new Form<>().setPageObject(this)
-            .setup(Form.class, e -> e.setName(getName() + " Form").setParent(this));
+                .setup(Form.class, e -> e.setName(getName() + " Form").setParent(this));
     }
 
-    private static Safe<String> currentPage = new Safe<>("Undefined Page");
+    private static final Safe<String> currentPage = new Safe<>("Undefined Page");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5482899)